### ISSUE TITLE
Would you give me some ideas about fine-tune stable animator on our  private dataset ?

### ISSUE THREAD

**Issue by Albertchamberlain** [Francis-Rings/StableAnimator#39]
as the title says, I want to fine-tune the models on our private datasets, for now, I have some ideas
1. fine-tune the total model
2. fine-tune the sub-model, such as the SVD， DWPose part

Do you agree with my proposal, and how to begin? 

Thx

**Comment by Francis-Rings** [comment]
Hi, the training components should involve the PoseNet, the face encoder, and the attention parts of U-Net in the SVD. If you have substantial GPU resources, you can finetune the whole U-Net in SVD instead of attention parts. Additionally, it is unnecessary to finetune the DWPose part.

**Comment by Albertchamberlain** [comment]
> Hi, the training components should involve the PoseNet, the face encoder, and the attention parts of U-Net in the SVD. If you have substantial GPU resources, you can finetune the whole U-Net in SVD instead of attention parts. Additionally, it is unnecessary to finetune the DWPose part.

Thx, got it, that is exactly what I want to know

**Comment by Francis-Rings** [comment]
I have just updated the README file. For the details of finetuning StableAnimator, please refer to the Model Training section.

**Comment by Albertchamberlain** [comment]
So, which data should I prepare? and what data or GT? I need to prepare the image and the pose series and face series ?

**Comment by Francis-Rings** [comment]
You should prepare several videos and extract the corresponding skeleton poses and face masks. For details on the dataset's file structure, please refer to the Model Training section.

**Comment by Albertchamberlain** [comment]
got it,thanks
